nixos/modules/services/web-apps/glitchtip.nix

```diff
@@ -171,7 +177,8 @@ in
 
           wants = [ "network-online.target" ];
           requires =
-            lib.optional cfg.database.createLocally "postgresql.target"
```

Comment:
I think besides the formatting issue, the postgresql target should not be removed